src/Serilog.Sinks.Elasticsearch/Sinks/ElasticSearch/ElasticSearchTemplateProvider.cs

```diff
@@ -44,97 +43,205 @@ public class ElasticsearchTemplateProvider
                 case AutoRegisterTemplateVersion.ESv5:
                     return GetTemplateESv5(settings, templateMatchString);
                 case AutoRegisterTemplateVersion.ESv6:
-                    return GetTemplateESv6(settings, templateMatchString);
+                    return GetTemplateESv6(null, null, settings, templateMatchString);
+                case AutoRegisterTemplateVersion.ESv7:
+                    return GetTemplateESv7(null, null, settings, templateMatchString);
                 case AutoRegisterTemplateVersion.ESv2:
                     return GetTemplateESv2(settings, templateMatchString);
```

Comment:
Can also be removed?

Reply:
Leaving this in for now, will follow up in a different PR